docs/overview/renderers/arm2d.rst

```diff
@@ -7,9 +9,9 @@ Microcontrollers**. It supports all Cortex-M processors ranging from
 Cortex-M0 to the latest Cortex-M85.
 
 Arm-2D accelerates LVGL9 with two modes: **Synchronous Mode** and
-**Asynchronous Mode**. 
-- When **Helium** and **ACI (Arm Custom Instruction)** are available, it is recommend
-  to use **Synchronous Mode** to accelerate LVGL. 
+**Asynchronous Mode**.
+- When **Helium** and **ACPI (Arm Custom Instruction)** are available, it is recommend
```

Comment:
ACI is an official term; please do not change it here.

Reply:
It was correct automatically by `typos`.